lib/gh-koenig/addon/components/gh-koenig.js

```diff
@@ -24,7 +24,6 @@ export default Component.extend({
 
     init() {
         this._super(...arguments);
-        this.container = document.querySelector('.gh-editor-container')[0];
```

Comment:
I looked and couldn't find any usage of `container` in any of the editor component files (js or hbs), so I assume this was used once and didn't get removed?